### DISCUSSION TITLE
Separate PostgreSQL as a separate language / platform from SQLite

### DISCUSSION THREAD

**Discussion by Kimmyera** [codewars/codewars.com#3255] [Ideas]
I continue to have to skip Kata because (for now) I am only interested in learning and practicing with pure SQL, whether with MySQL/SQLite or MS-SQL/SQL-Express functions to use. 

I keep finding varying differences between functions of PostGreSQL and SQLite to be too different from each other, to the point where I question why they even belong with each other with so many big differences like that, especially as you get into higher levels of Kyu difficulty. 
Also having to go out of the way to learn another set of functions and procedures when I would want to simply practice with 'regular' SQL at first, until the user is personally ready to transition to such.

**Comment by nourdev75536** [comment]
It means that you prefer to focus on learning SQL in its traditional and familiar forms, such as MySQL, SQLite, and MS-SQL/SQL-Express, without worrying about the significant differences between PostgreSQL and SQLite. You find these differences make it difficult to progress and learn effectively, and you prefer to start by practicing standard SQL before transitioning to databases that offer a different set of functions and procedures.
